Dockerfile

```diff
@@ -16,10 +16,10 @@ RUN dpkg --add-architecture i386
 
 RUN if [ "$USE_MLIR" = "ON" ] ; \
         then export ROCM_APT_VER=.apt_4.3.1;\
```

Comment:
Please help also bump the MLIR ROCm dependency as well.

There's nothing I'm aware of in MLIR that is incompatible with latest ROCm. Thanks!